instagram-app/src/PostsContainer/ComentSection/CommentSection.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./CommentSection.css";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      currentComment: ""
+    };
+  }
+  handleCommentChange = e => {
+    e.preventDefault();
+    this.setState({ currentComment: e.target.value });
+  };
+  handleCommentSubmit = e => {
+    e.preventDefault();
+    let commentList = this.state.comments;
+    let newComment = {
+      username: "GUEST01",
```

Comment:
Have you thought about using local storage to save an incoming user name on login? Something to think about implementing for this week.